mandatory/3-function-output.js

```diff
@@ -1,18 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
+  
 }
+// It generates a random number using Math.random() and it multiplies by 10
```

Comment:
Exactly correct.

:question: Could you perhaps explain what the output of `Math.random()` is? Did you happen to find any documentation on this to help explain it?

As a heads up - It's usually common to put comments above the code they are describing!